client/styles/components/_preview-nav.scss

```diff
@@ -0,0 +1,14 @@
+.preview-nav__editor-svg {
+  & svg {
+    width: #{44 / $base-font-size}rem;
```

Comment:
okay. i think i figured it out! i tried changing the width and height here from 44 to 22, and i think it looks better. i think we were comparing the size of different things! i meant the play/stop icon as in just the triangle/square icon, and i think that you thought i meant the icon + the circular button surrounding it.

Reply:
That must be it ! Fixed !